heapster.go

```diff
@@ -40,6 +40,7 @@ var (
 	argCacheDuration   = flag.Duration("cache_duration", 4*time.Minute, "The total duration of the historical data that will be cached by heapster.")
 	argUseModel        = flag.Bool("use_model", true, "When true, the internal model representation will be used")
 	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
+	argModelFrequency  = flag.Duration("model_frequency", 145*time.Second, "Frequency at which model will be updated. Applies only if use_model is true")
```

Comment:
It seems not: https://github.com/kubernetes/heapster/issues/642#issuecomment-146972283.